pubsub/.flake8

```diff
@@ -4,3 +4,9 @@ exclude =
   .git,
   *.pyc,
   conf.py
+
+ignore =
```

Comment:
I would *very* strongly prefer that we did not add these exceptions. No other project I maintain has needed to do this.